spooq/transformer/enum_cleaner.py

```diff
@@ -141,8 +151,16 @@ def transform(self, input_df):
                     .otherwise(F.when(F.col(column_name).isin(elements), substitute).otherwise(F.col(column_name)))
                     .cast(data_type),
                 )
+            elif mode == "NOT_DEFINED":
+                raise RuntimeError(
+                    f"Please provide a `mode` attribute!\n"
```

Comment:
also mention the two possible modes in this error message, e.g:
`Please provide a `mode` attribute ("allow" or "disallow")!`